packages/react/src/accordion/root/AccordionRoot.test.tsx

```diff
@@ -65,15 +65,15 @@ describe('<Accordion.Root />', () => {
       expect(trigger).to.have.attribute('aria-expanded', 'false');
       expect(queryByText(PANEL_CONTENT_1)).to.equal(null);
 
-      await user.pointer({ keys: '[MouseLeft]', target: trigger });
+      await user.click(trigger);
```

Comment:
Vitest doesn't have the `pointer` method but clicks should work the same way in these cases